### ISSUE TITLE
Possible new PKM app for you

### ISSUE THREAD

**Issue by pm64** [BluBloos/Obsidian-Time-Blocking#6]
Hi @BluBloos, I noticed you're archiving https://github.com/BluBloos/Obsidian-PDF due to disillusionment with the Obsidian ecosystem, and I suspect you'll be doing the same with this repo.

But before you do -- just wanted to bring the following project to your attention:

https://github.com/silverbulletmd/silverbullet

Might be worth looking into if you're looking for a change of scenery.

**Comment by BluBloos** [comment]
my goodness, thank you for recommending this! seems like it runs on the web, has collab features for two people in the vault, and is open source. Already better than Obsidian ...

**Comment by pm64** [comment]
Yes, I've been following it for a while. I think the developer had similar feelings about Obsidian when he started the project. Definitely has enormous potential. Enjoy!
